styles/twitch/catppuccin.user.css

```
@@ -604,6 +604,11 @@
     [style="color: rgb(245, 0, 155);"] {
       color: @pink !important;
     }
+
+    [style*="rgb(255, 255, 255)"] .chat-author__display-name,
+    [style*="rgb(255, 255, 255)"] .message-author__display-name {
+      color: #fff !important;
```

Using `@text` here would be preferable.
```suggestion
      color: @text !important;
```